packages/gestalt/src/Touchable.css.flow

```diff
@@ -6,9 +6,11 @@ declare module.exports: {|
   +'fullWidth': string,
   +'grab': string,
   +'grabbing': string,
+  +'inline': string,
```

Comment:
are we adding support for the `inline` prop?  if not - we should remove this (okay with keeping support for the prop btw to make the transition easier)

Reply:
Oops. I forgot to remove. No, we aren't adding support for `inline` prop since one can just wrap Touchable with `<Box display="inlineBlock">`. Thanks for catching this. I will also update the PR description.